godtools/FlowControllers/BaseFlowController.swift

```diff
@@ -105,6 +108,14 @@ class BaseFlowController: NSObject, UINavigationControllerDelegate {
                         dst?.view.transform = CGAffineTransform(translationX: 0, y: 0) },
                        completion: { finished in
                         _ = navigationController?.popViewController(animated: false) } )
+        
+        self.shouldRemoveMenu = false
+    }
+    
+    func removeMenuViewController() {
+        let menuPosition = (self.currentViewController?.navigationController?.viewControllers.count)! - 2
```

Comment:
@pablomarti rather than doing a count, wouldn't it be better to loop through the view controller stack and find one that `isKindOf: MainMenuController.self`? That would be safer than this if another view controller got pushed on to the stack somehow in the future.